src/hearings/containers/linked-hearings/linked-hearings.component.html

```diff
@@ -0,0 +1,11 @@
+<exui-app-header></exui-app-header>
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <a class="govuk-back-link" href="javascript:history.back()">Back</a>
+      <exui-linked-hearings-case-title [caseTitle]="caseTitle"></exui-linked-hearings-case-title>
```

Comment:
This can be passed caseId, caseName separately to the exui-linked-hearings-case-title, and let the sub component print the caseTitle.